src/main/java/dukeclasses/TaskManager.java

```diff
@@ -0,0 +1,183 @@
+package dukeclasses;
+
+import java.time.DateTimeException;
+import java.util.ArrayList;
+import java.time.LocalDate;
+
+/**
+ * Handles all the issues related to Tasks.
+ * Similar to TaskList
+ */
+public class TaskManager {
+
+    protected ArrayList<Task> listOfTasks;
+    protected Storage storage;
+
+    public TaskManager() {
+        listOfTasks = new ArrayList<>();
+        storage = new Storage();
+    }
+
+    /**
+     * loads existing data from data.txt.
+     */
+    public void loadExistingData() {
+
+        listOfTasks = storage.loadExistingData();
+    }
+
+    /**
+     * saves current data into data.txt
+     */
+    public void saveExistingData() {
+        storage.saveExistingData(listOfTasks);
+
+    }
+
+    /**
+     * Used for Level-9. find keyword that appears in each dukeClasses.Task in listOfTasks.
+     *
+     * @param keyword keyword to search
+     */
+    public void findTask(String keyword) {
+        System.out.println("Here are the matching tasks in your list:");
+        for (int i = 0; i < listOfTasks.size(); i++) {
+            if (listOfTasks.get(i).description.contains(keyword)) {
+                System.out.println("  " + (i + 1) +  ". " + this.listOfTasks.get(i).toString());
+            }
+        }
+    }
+
+    /**
+     * lists down all tasks.
```

Comment:
Do remember to capitalize the first word of your JavaDoc comments to ensure consistency!